WordPress/Classes/ViewRelated/Stats/Extensions/StatsViewController+JetpackSettings.swift

```diff
@@ -4,12 +4,11 @@ import UIKit
 extension StatsViewController {
 
     @objc func activateStatsModule(success: @escaping () -> Void, failure: @escaping (Error?) -> Void) {
-        guard let blog = blog,
-              let context = blog.settings?.managedObjectContext else {
+        guard let blog = blog else {
```

Comment:
Nitpick. This could be condensed to 
```suggestion
        guard let blog else {
```